round4/team-18.md

```diff
@@ -0,0 +1,106 @@
+20. As a student, I want to track how much time I spend on distracting versus non-distracting websites so that I can adjust my study habits for better outcomes.  
+21. As a user, I want to have a cute and visually appealing interface for my productivity tool, so that I am more inclined to use it regularly.  
+22. As a user, I want to receive analytics on my screen and webpage usage, so that I can understand and refine my browsing habits. 
+23. As a user, I want to interact with my Tamagotchi by feeding it or playing with it during breaks, so that I am encouraged to take healthy breaks and return to work refreshed. 
+24. As a user, I want to see a visual reminder of my progress towards my productivity goals, so that I am motivated to continue developing positive work habits. 
+
+## Notes
+
+## References & Inspiration 
+One existing product that we could draw inspiration from is "Tabagotchi," a very similar chrome extension tool. This tool lowers the health of the tamagotchi every time a new tab is opened, and the pet will die when too many tabs are closed. We could draw inspiration for visual and UI elements from this reference. 
+
+I also found an existing repo that displays just a simple tamagotchi in the browser, so we can draw inspiration from the basic setup of that project.
+
+## Technical Details
+
+### User Interface
+
+The extension will display a small sidebar that displays the tamagotchi design, its health bar, and a "settings" button.
+
+The "settings" button will take you to a website. This website will display the tamagotchi and its health bar, the list of off task and on task websites with options to add websites and delete websites, and the total nubmer of minutes spent on task and off task. 
+
+### API, Libraries, and Frameworks
+
+We plan to use the React framework to build our tool. 
+
+### Data Storage
+
+We need to store the current session's off task and on task time. We envision that we will store this data locally, so we do not need additional data storage. One of the TAs let us know that Chrome extensions have about 10mb of local storage, and part of their API (storage.local) that will help us manage that storage.
+
+## Project Management
+
+### Collaboration and Task Allocation
+
+- **Leader:** Clara
+- **Manager:** Laya
+- **Remaining Team Members:** [John, Via]
+
+  Clara: Help with the visual and UI design. 
+  Laya: Delegate tasks and make sure things are completed. 
+  John:  Help with overall project management and the software architecture. 
+  Via: Help with the data storage component. 
+
+We will use a shared GitHub respository, which we've already created, to share code.
+
+### Risks and Mitigation
+
+A potential risk is adding too many features and being unable to accomplish tasks becuase we are new to the project. One way to mitigate these risks is by ensuring that we get the basic functionality down before we move on to advanced features.

```

Comment:
One thing that might help here is prioritizing the features you've listed and then implementing those first. This would help you break your project into clearer milestones